compiler/dcalc/ast.mli

```diff
@@ -183,6 +184,10 @@ end
 
 module VarMap : Map.S with type key = Var.t
 
+val fv : expr Pos.marked -> unit VarMap.t
```

Comment:
`fv` should be renamed. What is the difference with `free_vars` ?

Reply:
Why not `free_vars_set` and `free_vars_list` ?

Reply:
done